pre_commit/main.py

```diff
@@ -92,11 +92,13 @@ def _add_run_options(parser: argparse.ArgumentParser) -> None:
     parser.add_argument('--verbose', '-v', action='store_true', default=False)
     parser.add_argument(
         '--origin', '-o',
-        help="The origin branch's commit_id when using `git push`.",
+        help="The origin branch's commit_id when using `git push`. "
+        + 'The ref of the previous HEAD when using `git checkout`.',
```

Comment:
instead of ugly `+` use parens to continue lines (see below there's a few examples -- search for `help=(`)